scripts/release.ts

```diff
@@ -86,11 +109,32 @@ async function bumpVersions() {
     console.log(c.yellow('No version changes made. Exiting...'));
     process.exit(1);
   }
-  console.log(c.green('Bumped package versions.'));
+  console.log(c.green('Bumped package versions. Bumping dependencies next...'));
+
+  getPackages().forEach((pack) => {
+    const [packageName] = Object.keys(pack);
+    const [packageLocation] = Object.values(pack);
+    const jsonLocation = path.join(
+      root,
+      packageLocation.split('file:')[1].replaceAll('../', ''),
+      'package.json'
+    );
+    const json = require(jsonLocation);
+    const updatedJson = updateDSVersion(json);
+    writeJson(jsonLocation, updatedJson);
+    console.log(
+      c.green(`Bumped ${packageName} to @cmsgov/design-system@${newDesignSystemVersion()}.`)
+    );
+  });
 
   // Update versions.json
   const currentVersionsByPackage = updateVersions();
+  // Clean up our staged files
+  sh('git reset ./packages/docs/package.json');
+  sh('git checkout ./packages/docs/package.json');
   sh('git add -u');
+  // Delete lingering package-lock.json
+  sh('rm ../package-lock.json');
```

Comment:
Is this line needed? I got the following error when running locally:
```
rm: ../package-lock.json: No such file or directory
Command failed: rm ../package-lock.json
```

Maybe a try/catch block would suffice or a check to see whether the file exists before removing it?

Reply:
Good idea. I also got the proper directory setup with the latest push.